tests/feed/attachments_base_test.php

```diff
@@ -31,13 +31,22 @@ public function setUp()
 
 		$this->filesystem = new \phpbb\filesystem();
 		$config = new \phpbb\config\config(array());
+		$path_helper = new \phpbb\path_helper(
+			new \phpbb\symfony_request(
+				new phpbb_mock_request()
+			),
+			new \phpbb\filesystem\filesystem(),
```

Comment:
Here you can reuse $this->filesystem instead of instance it again

Reply:
Good point!